src/ElseCanRender.tsx

```diff
@@ -1,5 +1,12 @@
-import React from 'react';
-import { ICanRenderProps } from './interfaces/ICanRenderProps';
-import { CanRenderBase } from './CanRenderBase';
+import React, { Suspense } from 'react';
+import { ICanRenderProps } from './CanRenderBase';
 
-export const ElseCanRender = (props: ICanRenderProps) => <CanRenderBase {...props} />;
+const CanRenderBase = React.lazy(() => import('./CanRenderBase'));
```

Comment:
Here is the same question, why don't use the old approach?

Reply:
Solved.